### PR TITLE
Add button html file

### PR DESCRIPTION
**Pull request by Blackbaud-AdamHickey**
Add button html file that raises the constituent modal when the button click callback is called.

### CODE REVIEW

File: BlackbaudAddinHelloWorld/HelloWorldButton.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html>
+<head>
+	<meta charset="utf-8" />
+	<title></title>
+	<script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/@blackbaud/sky-addin-client@1.0.2/bundles/sky-addin-client.umd.min.js"></script>
+	<script src="HelloWorldButton.js"></script>
+</head>
+<body>
+	<p>Hello world button</p>
```
Comment (Blackbaud-BenLambert) [inline BlackbaudAddinHelloWorld/HelloWorldButton.html]: I don't think this is needed, right?  There is no UI from the HTML that is rendered, just used to get the script downloaded

File: BlackbaudAddinHelloWorld/HelloWorldButton.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  // BBSkyAddinClient is global here.
+  var client = new BBSkyAddinClient.AddinClient({
+      callbacks: {
+          init: function (args) {
+              // inform the host page that the add-in is ready to be shown
+              args.ready({
+                  showUI: true,
+                  title: 'Add customer',
+                  buttonConfig: { style: 1 }
```
Comment (Blackbaud-BenLambert) [inline BlackbaudAddinHelloWorld/HelloWorldButton.js]: can you use the enum here?

**Review by Blackbaud-BenLambert — CHANGES_REQUESTED** [review]

File: BlackbaudAddinHelloWorld/HelloWorldButton.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<title></title>
+	<script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/@blackbaud/sky-addin-client@1.0.2/bundles/sky-addin-client.umd.min.js"></script>
```
Comment (Blackbaud-BenLambert) [inline BlackbaudAddinHelloWorld/HelloWorldButton.html]: check the version here - should this be 1.0.7?

File: BlackbaudAddinHelloWorld/HelloWorldButton.html (hunk 1 above)
Comment (Blackbaud-AdamHickey) [reply BlackbaudAddinHelloWorld/HelloWorldButton.html]: It's not necessary.

File: BlackbaudAddinHelloWorld/HelloWorldButton.js (hunk 2 above)
Comment (Blackbaud-AdamHickey) [reply BlackbaudAddinHelloWorld/HelloWorldButton.js]: Yeah good call. Updated

**Review by Blackbaud-BenLambert — APPROVED** [review]

**Pull request merged**